resources/views/people/index.blade.php

```diff
@@ -42,6 +42,16 @@
                         <a href="/people">{{ trans('people.people_list_clear_filter') }}</a>
                       </p>
                   @endif
+                  @if ($tagLess)
+                      <p class="clear-filter">
+                        {!! trans('people.people_list_filter_tag') !!}
```

Comment:
The syntax `{!!` is for unescaped echo. Perhaps it's better here to use `{{` for html escape.